src/Elsa.Dashboard/src/components/common-properties/he-text-group-property.tsx

```diff
@@ -167,6 +168,11 @@ export class HeTextGroupProperty implements ISortableSharedComponent, IDisplayTo
       let maximiseIconStyle = !isMinimised ? this.hiddenValue : this.displayValue;
       let displayGroupStyle = isMinimised ? this.hiddenValue : "";
 
+      const conditionSyntax = SyntaxNames.JavaScript;
+      const conditionLanguage = mapSyntaxToLanguage(conditionSyntax);
+      const conditionExpression = textGroup.expressions[TextActivityOptionsSyntax.Condition];
```

Comment:
I can't see any setting up of this expression value in the onAddGroupClick() method - so I presume this doesn't complain that the key doesn't exist in the expression dictionary?

Reply:
Fixed.